Umbraco Cloud release notes for August (#5374)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2023-08-releasenotes.md

# August 2023

## Key Takeaways
* **Australia region hosting** -  Umbraco Cloud expands its regional hosting offerings with the addition of _Australia East_, catering to a wider range of hosting preferences.
* **Visualization of Events on “Availability & Performance”** - Elevate project insights with Umbraco Cloud's Event Visualization. Track critical events and their impact on your project's journey.
* **Improved trial flow for Umbraco Cloud** - The trial flow has undergone a significant visual upgrade, now offering a more engaging and intuitive experience.

## Australia region hosting

> As the interest in adopting Umbraco Cloud as the preferred development and hosting platform for Umbraco projects continues to grow, there is a corresponding surge in demand for expanded regional hosting choices. This demand is particularly pronounced given factors like privacy regulations and data governance standards, as the physical location of data holds increasing significance.

Reviewer: we have a styleguide in place to make the documentation more readable and one of the factors is to [write sentences shorter than 25 words.](https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences) therefore I am suggesting a rephrasing of the 1st sentence:
```suggestion
There is an interest increase in expanding the regional hosting choices for Umbraco Cloud as the preferred development and hosting platform for Umbraco projects. This demand is particularly pronounced given factors like privacy regulations and data governance standards, as the physical location of data holds increasing significance.
```

> The availability of **_regional hosting_** options empowers you to pinpoint the exact location for hosting and storing your Umbraco Cloud project, encompassing all its environments and data with a significant improvement in performance for users close to the selected hosting center.

Reviewer: to keep it to 25 words per sentence:
```suggestion
The availability of **_regional hosting_** options empowers you to pinpoint the exact location for hosting and storing your Umbraco Cloud project. It also encompasses all its environments and data with a significant improvement in performance for users close to the selected hosting center.
```

![Australia East cloud region](images/AusEast.png)

Another significant stride for Umbraco Cloud is the introduction of the East Australia regional hosting option, marking the fourth supported region. This addition is met with great enthusiasm by professionals within this region and those collaborating with clients here

Read the entire [blog post](https://umbraco.com/blog/umbraco-cloud-expands-with-regional-hosting-option-in-australia/) for more information.

## Visualization of Events on “Availability & Performance”

> Umbraco Cloud introduces a pragmatic enhancement – Event Visualization on the "Availability & Performance" dashboard. This feature seamlessly incorporates event monitoring into the SVG visualization for each metrics panel.

Reviewer: we also need to [define acronyms](https://docs.umbraco.com/welcome/contribute/style-guide#defined-acronyms) such as SVG
```suggestion
Umbraco Cloud introduces a pragmatic enhancement – Event Visualization on the "Availability & Performance" dashboard. This feature seamlessly incorporates event monitoring into the Scalable Vector Graphics (SVG) visualization for each metrics panel.
```

![Event 1](images/Event1.png)

Gain insight into critical project events, such as restarts, automatic and manual upgrades, updates involving environment deployments, and changes in plans. This concise display allows for a comprehensive understanding of the events affecting your project.

![Event 2](images/Event2.png)

> Stay informed about pivotal alterations, ranging from routine restarts to intricate deployments and upgrades. Whether it's a system restart, an environment deployment, or an upgrade, you'll have gained more of the information you need to assist you in potential troubleshooting, make informed decisions, and ensure smooth project management.

Reviewer: to keep the sentences to 25 words:
```suggestion
Stay informed about pivotal alterations, ranging from routine restarts to intricate deployments and upgrades. Whether it's a system restart, an environment deployment, or an upgrade, you'll have gained more of the information you need. This information will assist you in potential troubleshooting, make informed decisions, and ensure smooth project management.
```
